subprojects/docs/src/snippets/java/customDirs/groovy/build.gradle

```diff
@@ -52,9 +52,17 @@ reporting.baseDir = "my-reports"
 testResultsDirName = "$buildDir/my-test-results"
 
 tasks.register('showDirs') {
+    def rootDirPath = project.rootDir.toPath()
+
+    def testReportDir = project.getExtensions().getByType(JavaPluginExtension.class).getTestReportDir().get()
```

Comment:
❓ What's wrong with the original expression `project.reportsDir`?

Reply:
@bamboo - good point, nothing wrong with original project.reportsDir, I got carried away trying to align the groovy & kotlin scripts : )

I've pushed a fix to my branch, to use the original project.reportsDir (and also undid the change I had made to the test output for the same)